CSS/style.css

```diff
@@ -35,31 +35,31 @@ a {
     height: calc(100vh + 24px) ;
     width: 100%;
 }
-.navigate {
+.navigatePanel {

```

Comment:
in css there is class naming convention most of developers use.
Read here: https://gist.github.com/whizark/92ed70d90d262a44db84#phrase-and-compound-word